### DISCUSSION TITLE
colors in `tar_make()` output are gone?

### DISCUSSION THREAD

**Discussion by ginolhac** [ropensci/targets#302] [Help] [answered]
Hello,

I quick note about `tar_make()` which used to be colored with green check icon, blue dots or red when errors.
Now I have all of the same colors.

```
✓ skip target cells
✓ skip target contrasts
✓ skip target annotation
✓ skip target dds
✓ skip target fc
✓ skip branch sub_dds_3078b1e0
✓ skip branch sub_dds_d05c5da7
✓ skip branch sub_dds_c60d7096
✓ skip target tpm
✓ skip branch deg_lrt_4c175699
✓ skip branch deg_lrt_89f78360
✓ skip branch deg_lrt_27ca171e
✓ skip branch deg_wld_9ea18863
✓ skip branch deg_wld_224dd40f
✓ skip branch deg_wld_f2e27deb
```

 I tried to `options(crayon.enable = TRUE)` (if it was disabled by default recently) but same monocolor. 
I haven't seen a change that could explain that. Tool is entirely functional, but I kind of liked the previous colourful output.

Many thanks again for such a nice package!

**Comment by wlandau** [comment, marked as answer]
Depends on your terminal. Seems to work fine in the RStudio IDE but not in the shell. Should work again in https://github.com/ropensci/targets/commit/09f57470fcf364f749a99ca2d2aa2b8cece4e217. It should have worked before, I do not know what changed.

**Reply by wlandau** [reply]
Update: the default is now to enable `crayon` if and only if the parent R session is interactive. (`crayon` in non-interactive sessions produces messy log files.)

**Reply by ginolhac** [reply]
Hi, 
thanks for the fast reply! I can confirm that in the console, colors are back!
![image](https://user-images.githubusercontent.com/710181/107053208-dafefd00-67ce-11eb-8433-2d11cb53a2f5.png)

but in the Build pane I usually use nope
![image](https://user-images.githubusercontent.com/710181/107053283-ee11cd00-67ce-11eb-84b8-71f2294f36d7.png)

but that's a RStudio IDE problem (using here 1.4.1103) not `targets`. Thanks a lot!

**Reply by wlandau** [reply]
Thanks for confirming. 

The build pane is non-interactive, so `targets` disables `crayon` there. Log files and R Markdown reports and `reprex`es do not properly render colors in non-interactive sessions, and I want to avoid printing messy terminal codes in those situations.

**Reply by ginolhac** [reply]
Perfect!! I am adding `options(crayon.enabled = TRUE)` to the `_targets.R` and now I have the colors also in the Build pane :+1: 

![image](https://user-images.githubusercontent.com/710181/107055387-8a3cd380-67d1-11eb-8784-a5d97ff057a9.png)

now the text is in white compared the previous red which is actually even better. Thanks a lot! I love it  :heart:
